module/src/hooks/form/form.utils.ts

```diff
@@ -77,3 +78,14 @@ export function isArrayValue(value: any, attemptedAction: string): value is any[
   }
   return true;
 }
+
+/**
+ * Detects whether the incoming initial form data is a callback or an object and casts appropriately.
+ * @param initialData Either some initial form data, or a function that returns initial form data.
+ * @returns true if param is a function, also casts appropriately.
+ */
+export function initialDataIsCallback<TData extends object>(
+  initialData?: TData | InitialDataFunction<TData>
```

Comment:
change this to theDataThatIsInitial

Reply:
or preferably `dataOnlyItIsTheDataThatIsInitialAsInTheFirstData`

Reply:
Resolving due to general clownery